namelists/echam/6.3.04p1/cmip6/namelist.echam

```diff
@@ -0,0 +1,139 @@
+&parctl
+	nproca=24
+	nprocb=48
+/
+&runctl
+	NPROMA=72
+	OUT_FILETYPE=2
+	OUT_ZTYPE=0
+	RERUN_FILETYPE=4
+	LTIMER=.false.
+	LMIDATM=.true.
+	LFRACTIONAL_MASK=.false.
+	!LHD=.true.
+	LCOUPLE=.true.
+	LCOUPLE_CO2=.true.
+	!LIPCC=.true.
+	LAMIP=.false.
+	PUTOCEAN=1,'days','last',0
+	GETOCEAN=1,'days','last',0
+	PUTDATA=24,'hours','last',0
+	default_output=.true.
+	TRIGFILES=12,'months','exact',0
+	LDEBUGEV=.false.
+	!lmeltpond=.false.
+	out_expname=From2017in2019in2017Climatessp3703
+	dt_stop=2019,10,01
+	dt_resume=2019,07,01
+	lresume=.true.
+	out_datapath='/work/ba1138/a270148/esm-experiments/AWICM/From2017in2019in2017Climatessp3703/work//'
+	delta_time=200
+	putrerun=3,'months','first',0
+/
+&submdiagctl
+	vphysc_lpost = .false.
+/
+&mvstreamctl
+	interval = 1, 'months', 'last', 0
+	target = 'spim'
+	source = 'sp'
+	variables = 'st:mean', 'svo:mean', 'lsp:mean', 'sd:mean'
+/
+&mvstreamctl
+	interval = 1, 'months', 'last', 0
+	target = 'glim'
+	source = 'gl'
+	variables = 'xl:mean', 'xi:mean'
+/
+&mvstreamctl
+	interval = 1, 'months', 'last', 0
+	target = 'aclcim'
+	source = 'g3b'
+	variables = 'aclc:mean'
+/
+&set_stream_element
+	stream = 'aclcim'
+	name = 'aclc'
+	code = 162
+	lpost = 1
+/
+&set_stream
+	stream = 'jsbach'
+	lpost = 0
+/
+&set_stream
+	stream = 'land'
+	lpost = 0
+/
+&set_stream
+	stream = 'veg'
+	lpost = 0
+/
+&set_stream
+	stream = 'yasso'
+	lpost = 0
+/
+&mvstreamctl
+	interval = 1, 'months', 'last', 0
+	target = 'g3bim'
+	source = 'g3b'
+	variables = 't2min:min', 't2max:max', 'topmax:mean'
+/
+&mvstreamctl
+	interval = 1, 'days', 'last', 0
+	target = 'glday'
+	source = 'gl'
+	variables = 'q:mean'
+/
+&mvstreamctl
+	target = 'g3bday'
+	interval = 1, 'days', 'last', 0
+	source = 'g3b'
+	meannam = 'tslm1', 'tsi'
+/
+&mvstreamctl
+	interval = 1, 'days', 'last', 0
+	target = 'g3bid'
+	source = 'g3b'
+	variables = 'u10:mean', 'v10:mean', 'temp2:mean', 'relhum:mean',
+	'dew2:mean', 'ws:mean', 'sn:mean', 'wimax:max', 't2min:min', 't2max:max'
+/
+&mvstreamctl
+	interval = 1, 'days', 'last', 0
+	target = 'jsbid'
+	source = 'jsbach'
+	variables = 'layer_moisture:mean'
+/
+&mvstreamctl
+	interval = 6, 'hours', 'last', 0
+	target = 'sp6h'
+	source = 'sp'
+	variables = 'st:mean', 'svo:mean', 'lsp:mean', 'sd:mean'
+/
+&mvstreamctl
+	interval = 1, 'hours', 'last', 0
+	target = 'g3b1hi'
+	source = 'g3b'
+	variables = 'u10:inst', 'v10:inst', 'wimax:max'
+/
+&dynctl
+	ENSTDIF = 1.
+/
+&submodelctl
+	lmethox = .true.
+	lco2 = .false.
+/
+&radctl
+	iaero = 9
+	io3 = 4
+	isolrad = 1
+	ico2 = 4
+	ighg = 1
+	icfc = 4
+	ich4 = 4
+	in2o = 4
```

Comment:
I am not sure about these settings here. If they are what is recommended by MPI-Met, then it's good... One could double check using the MPI Infrastructure toolkit `mkexp` and see values are used there.